### ISSUE TITLE
WorkflowCore.Abstractions ?

### ISSUE THREAD

**Issue by glucaci** [danielgerlag/workflow-core#401]
Can we create an Abstraction package?

It will be useful to have the Host and the Workflows in separate projects. Than in Workflows project can reference only the Abstractions package.

**Comment by danielgerlag** [comment]
I don't have any plans for that since, Workflow Core is the only implementation of these interfaces.

**Comment by glucaci** [comment]
Yep, I saw... I tough more on separating the workflow definitions dependencies from the workflow executor somehow.

**Comment by danielgerlag** [comment]
That is a very good point

**Comment by wfcore-manager** [comment]
Closing as not planned — this has been open for several years without recent progress. Please open a new issue if this is still relevant on a current Workflow Core release.
